data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report11_covid_concept_no_suppression.py

```diff
@@ -29,6 +30,11 @@
 project_id: str = ""  # identifies the project where datasets are located
 post_deid_dataset: str = ""  # the deid dataset
 run_as = ""
+
+# +
+#project_id = "aou-res-curation-prod"
```

Comment:
These parameters should be removed, we do not want our dataset and project names in the git history.